A32NX/html_ui/Pages/A32NX_Core/A32NX_APU.js

```diff
@@ -6,12 +6,31 @@ class A32NX_APU {
         console.log('A32NX_APU init');
         SimVar.SetSimVarValue("L:APU_FLAP_OPEN", "Percent", 0);
         this.lastAPUBleedState = -1;
+
+        // FIX FOR 1.8.3 UPDATE
+        SimVar.SetSimVarValue("L:A32NX_FUEL_FIX_APPLIED", "Enum", -1);
     }
     update(_deltaTime) {
-        const currentAPUMasterState = SimVar.GetSimVarValue("FUELSYSTEM VALVE SWITCH:8", "Bool");
+        const currentAPUMasterState = SimVar.GetSimVarValue("A:FUELSYSTEM VALVE SWITCH:8", "Bool");
         const apuFlapOpenPercent = SimVar.GetSimVarValue("L:APU_FLAP_OPEN", "Percent");
         const APUPctRPM = SimVar.GetSimVarValue("APU PCT RPM", "percent");
 
+        //////////////////////////
```

Comment:
Wouldn't it be better to check the `center_qty` first and only if it's below 40 gallons you can apply the fix?

Reply:
I believe that is what we are doing

Reply:
Yes, but all those expressions are moot if you find out that center_qty is greater than 40. So you should check center_qty first as it seems like the primary condition for this fix.